org.darkdimension.rue.json

```diff
@@ -0,0 +1,53 @@
+{
+	"id": "org.darkdimension.rue",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "20.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "rue",
+	"modules": [
+		"shared-modules/glew/glew.json",
+		"shared-modules/glu/glu-9.json",
+		{
+			"name": "avdl",
+			"buildsystem": "simple",
+			"build-commands": [
+				"make prefix=",
```

Comment:
shouldn't prefix be `${FLATPAK_DEST}` ?

Reply:
That is correct, I missed it and passed it during the installation step only. It is fixed now to be part of `prefix=`